tests/Optimizer/Transformer/MinifyHtmlTest.php

```diff
@@ -5,11 +5,13 @@
 use AmpProject\Dom\Document;
 use AmpProject\Optimizer\Configuration\MinifyHtmlConfiguration;
 use AmpProject\Optimizer\Error\InvalidJson;
+use AmpProject\Optimizer\Error\MissingPackage;
```

Comment:
Again, these should use relative namespaces:
```suggestion
use AmpProject\Optimizer\Error;
```